### PR TITLE
chore: add more cli tests

### PR DESCRIPTION
**Pull request by deer**
closes https://github.com/netzo/netzo/issues/96

I will pull the remote part (second bullet point) into a new issue, since this was already a lot of work and I don't want it to get any larger.

### CODE REVIEW

File: lib/cli/tests/help.test.ts
Comment (deer) [inline lib/cli/tests/help.test.ts:1]: pull `Deno.test` into the file as a best practice, so that each individual test can easily be debugged via the ui

File: deno.jsonc [hunk 1]
```diff
@@ -46,12 +46,12 @@
...
-    "preact/": "https://esm.sh/v135/preact@10.19.2/",
-    "react": "https://esm.sh/v135/preact@10.19.2/compat",
-    "react-dom": "https://esm.sh/v135/preact@10.19.2/compat",
-    "@preact/signals": "https://esm.sh/v135/*@preact/signals@1.2.1",
-    "@preact/signals-core": "https://esm.sh/v135/*@preact/signals-core@1.5.0",
+    "preact": "https://esm.sh/v135/preact@10.19.3",
+    "preact/": "https://esm.sh/v135/preact@10.19.3/",
+    "react": "https://esm.sh/v135/preact@10.19.3/compat",
+    "react-dom": "https://esm.sh/v135/preact@10.19.3/compat",
+    "@preact/signals": "https://esm.sh/v135/*@preact/signals@1.2.2",
```
[the remark this answers is not in the document]
Comment (deer) [reply deno.jsonc:53]: specifically `preact/signals`

File: lib/cli/tests/init_test.ts [hunk 2]
```diff
@@ -23,7 +25,125 @@ Deno.test("CLI init and task execution", async () => {
...
+      "task",
+      "test",
+      "--no-check",
+    ],
+    cwd: tmpDirName,
+  }).spawn();
+  await testProcess.status;
+
+  await retry(() => Deno.remove(tmpDirName, { recursive: true }));
+});
```
Comment (deer) [inline lib/cli/tests/init_test.ts:97]: This is a decent step forward since it asserts that the crm template:
* has no formatting issues
* has no linting issues
* can start AND it even passes the very minimal test (which just verifies the name appears in a certain spot). i'll log an issue to add a few more tests here, since users of netzo might appreciate having more example tests to get them started so they can learn how to write their own tests

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -25,7 +25,7 @@ jobs:
         run: deno fmt --check
 
       - name: Test
-        run: deno test -A lib/cli/tests/init_test.ts
+        run: deno test -A lib/cli/tests/
```
[the remark this answers is not in the document]
Comment (miguelrk) [reply .github/workflows/ci.yml:28]: Hahaha making progress!

File: lib/cli/tests/help.test.ts
Comment (miguelrk) [reply lib/cli/tests/help.test.ts:1]: Interesting... Good to now!

File: lib/cli/tests/init_test.ts (hunk 2 above)
Comment (miguelrk) [reply lib/cli/tests/init_test.ts:97]: Totally agree, it's very nice to have this in place. I appreciate also the tidiness of the tests 👌🏼. Looking forward to tackle #86

File: templates/minimal/fresh.config.ts
Comment (miguelrk) [inline templates/minimal/fresh.config.ts:1]: Is there a reason for the `fresh.config.ts` file? Might have slipped through @deer

Comment (deer) [reply templates/minimal/fresh.config.ts:1]: Due to https://github.com/denoland/fresh/blob/main/src/dev/cli.ts wanting a config. If we're planning on moving towards a more fresh-centric approach, it's probably necessary to have the file at some point, so might as well add it now.

**Comment by miguelrk** [comment]
Thanks for this reed, looking pretty solid 👌🏼 . I'd just have on nit, I see there's `help.test.ts`, `init_test.ts` under `lib/cli/tests/`, and then `test_utils.ts` and `test.ts` for both templates. Would you mind using `.` rather than `_` going forward? 

We adopted this convention primarly because of vscode's [file nesting feature](https://code.visualstudio.com/updates/v1_67#_explorer-file-nesting) plus the [`antfu.file-nesting`](https://github.com/antfu/vscode-file-nesting-config#file-nesting-config-for-vs-code) vscode extension, which I would recommend a lot.

So basically renaming `init_test.ts` to `init.test.ts` and `test_utils.ts` to `test.utils.ts`.

**Review by miguelrk — APPROVED** [review]
Thanks!
